packages/theme/src/cli/utilities/theme-environment/proxy.ts

```diff
@@ -254,6 +269,16 @@ function proxyStorefrontRequest(event: H3Event, ctx: DevServerContext) {
       redirect: 'manual',
     },
     async onResponse(event, response) {
+      if (
+        !event.path.startsWith(EXTENSION_CDN_PREFIX) &&
+        !event.path.startsWith(VANITY_CDN_PREFIX) &&
+        !event.path.endsWith('.js')
+      ) {
+        if (shouldLog(event)) {
+          logRequestLine(event, response)
+        }
+      }
```

Comment:
I believe we could just call `logRequestLine` here (and then move the `shouldLog` logic to `log-request-line.ts`).

Also, in the `log-request-line.ts`, we may add some unit tests to clarify our expectations about when we should log and when we shouldn't :)